optaplanner-examples/src/main/java/org/optaplanner/examples/common/experimental/ExperimentalConstraintCollectors.java

```diff
@@ -162,15 +162,17 @@
      * @param startMap Maps the fact to its start
      * @param endMap Maps the fact to its end
      * @param <A> type of the first mapped fact
-     * @param <C> type of the fact endpoints
+     * @param <PointType_> type of the fact endpoints
```

Comment:
`differenceFunction` is undocumented. Focus on the arguments to that function. Is the first one expected to be smaller than the other? What should happen if that's not the case?